### PR TITLE
Revert rollback of "add assistiveTechnologyEnabled to window"

### PR DESCRIPTION
**Pull request by jonahwilliams**

### CODE REVIEW

**Comment by jonahwilliams** [comment]
cc @amirh I havent been able to reproduce that error locally, what build configuration did it occur with?

File: shell/platform/darwin/ios/platform_view_ios.h [hunk 1]
```diff
@@ -57,10 +57,17 @@ class PlatformViewIOS final : public HeadlessPlatformViewIOS {
   // |shell::PlatformView|
   void SetSemanticsEnabled(bool enabled) override;
 
+  // |shell::PlatformView|
+  void SetAssistiveTechnologyEnabled(bool enabled) override;
+
+  // |shell::PlatformView|
+  void HandlePlatformMessage(
```
Comment (jonahwilliams) [inline shell/platform/darwin/ios/platform_view_ios.h]: This was teh cause, a bady copy paste 🥇

**Comment by jonahwilliams** [comment]
I copy-pasted code wrong and moved an override in platform_view_ios that already existed in the base class

**Comment by amirh** [comment]
It happened on the mac bot, sending you the link to the log

**Comment by jonahwilliams** [comment]
I got the error when I built with a debug_sim locally, removing the line of code I commented fixed it.  Bad copy paste

**Pull request merged**